__tests__/docker-compose-fixture.yml

```diff
@@ -1,4 +1,3 @@
-version: '3'
```

Comment:
docker was generating some deprecation message about "version", which was failing the logs test.
I just removed the version, but can also put it back and fix the test assertion instead if it matters to you.